tests/karate/src/test/java/fe/RollCall/rollCallCreation.feature

```diff
@@ -0,0 +1,31 @@
+@env=android,web
+Feature: Create RollCall
+
+  Background:
+    # Call the lao creation util that is front-end agnostic
+    * call read('classpath:fe/utils/simpleScenarios.feature@name=create_lao')
+
+    # Call the roll call ccreation util that is front-end dependant
+    * def rc_page_object = 'classpath:fe/utils/<env>.feature@name=create_roll_call'
+    * replace rc_page_object.env = karate.env
+    * call read(rc_page_object)
+
+  Scenario: Creating a Roll Call send right message to backend and displays element
+    Given backend.clearBuffer()
+    And backend.setRollCallCreateMode()
```

Comment:
I see everywhere these `setXMode()`, what do they do ? I'm a bit confused here and I can't seem to find the related code.